core/store/migrations/0058_direct_request_whitelist.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up58 = `
+ALTER TABLE direct_request_specs ADD COLUMN requesters TEXT; 
```

Comment:
Should that be `text[]`?

Reply:
I just re-used a pre-existing datatype and this was how it worked before, not perfect maybe but its battle-tested and works.

Reply:
ah good to know